src/settings-ui/Settings.UI/Flyout/LaunchPage.xaml

```diff
@@ -139,7 +139,8 @@
                     Padding="8"
                     x:Uid="SettingsBtn"
                     Click="SettingsBtn_Click"
-                    Style="{StaticResource FlyoutButtonStyle}">
+                    Style="{StaticResource FlyoutButtonStyle}"
+                    Foreground="{ThemeResource TextFillColorPrimary}">
```

Comment:
This is necessary because the settings icon does not change color with the theme until interacted with. Adding this forces the AnimatedIcon to use the appropriate foreground color.